ItalianTranslation_PlanetCoaster.xml

```diff
@@ -3019,188 +3019,188 @@
-		<entry key="TrackElementName_FrictionWheelLift_VNTR" translation="Friction Wheel Lift"/>
-		<entry key="TrackElementName_TireLaunch_Booster_VNTR" translation="Drive Tyre Launch"/>
-		<entry key="TrackElementTool_FrictionWheelLift_VNTR" translation="&lt;font color='accent'&gt;Friction Wheel Lift&lt;/font&gt;&#10;Propels coasters up a slope or along the track at moderate speeds.&#10;&#10;This track element can be used before the Drive Tyre Launch to create a dual stage launch."/>
-		<entry key="TrackElementTool_TireLaunch_Booster_VNTR" translation="&lt;font color='accent'&gt;Drive Tyre Launch&lt;/font&gt;&#10;Propels coasters along the track at a faster speed than the standard Friction Wheel Lift.&#10;&#10;This track element can be used after the Friction Wheel Lift to create a dual stage launch."/>
+		<entry key="Coaster_VNTR_Desc" translation="Coaster lanciato a pneumatici che include due differenti meccanismi a pneumatico per una velocità di lancio variabile.&#10;25° Anniversario Frontier"/>
+		<entry key="TrackElementDesc_FrictionWheelLift_VNTR" translation="Spinge i coaster su una salita o lungo un tracciato a velocità moderata.&#10;&#10;Questo elemento può essere utilizzato prima di un lancio pneumatico per creare un lancio a due tempi."/>
+		<entry key="TrackElementDesc_TireLaunch_Booster_VNTR" translation="Spinge i coaster lungo un tracciato ad una velocità maggiore rispetto al sollevamento a ruota di attrito standard.&#10;&#10;Questo elemento può essere utilizzato dopo il sollevamento a ruota di attrito per creare un lancio a due tempi."/>
+		<entry key="TrackElementName_FrictionWheelLift_VNTR" translation="Sollevamento a Ruota di Attrito"/>
+		<entry key="TrackElementName_TireLaunch_Booster_VNTR" translation="Lancio Pneumatico"/>
+		<entry key="TrackElementTool_FrictionWheelLift_VNTR" translation="&lt;font color='accent'&gt;Sollevamento a Ruota di Attrito&lt;/font&gt;&#10;Spinge i coaster su una salita o lungo un tracciato a velocità moderata.&#10;&#10;Questo elemento può essere utilizzato prima di un lancio pneumatico per creare un lancio a due tempi."/>
+		<entry key="TrackElementTool_TireLaunch_Booster_VNTR" translation="&lt;font color='accent'&gt;Lancio Pneumatico&lt;/font&gt;&#10;Spinge i coaster lungo un tracciato ad una velocità maggiore rispetto al sollevamento a ruota di attrito standard.&#10;&#10;Questo elemento può essere utilizzato dopo il sollevamento a ruota di attrito per creare un lancio a due tempi."/>
 		<!--C10_Facilities-->
-		<entry key="ShopItem_Balloon_SpecialEffect" translation="SpecialEffect Balloon"/>
-		<entry key="ShopItem_Hat_Baseball_SpecialEffect" translation="SpecialEffect Baseball Cap"/>
+		<entry key="ShopItem_Balloon_SpecialEffect" translation="Palloncino EffettiSpeciali"/>
+		<entry key="ShopItem_Hat_Baseball_SpecialEffect" translation="Cappellino Baseball EffettiSpeciali"/>
```

Comment:
Penso che SpecialEffect sia l'associazione, non andrebbe tradotto, potresti rimetterlo in inglese?

Reply:
Scusa , non mi sono arrivate le notifiche, correggo e rimando